app/src/main/res/layout/horizontal_itemlist_item.xml

```diff
@@ -15,6 +15,8 @@
         android:layout_height="wrap_content"
         android:foreground="?android:attr/selectableItemBackground"
         android:clickable="true"
+        app:cardElevation="0dp"
+        android:layout_margin="1px"
```

Comment:
There is another card inside that should get the margin. Like this, there is no border.